bin/capsule-remove

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+echo ""
+echo "WARNING: This script will erase many packages and config files."
+echo "Important packages such as the following will be removed:"
+echo ""
+echo "  * httpd (apache)"
+echo "  * mongodb"
+echo "  * tomcat"
+echo "  * puppet"
+echo "  * ruby"
+echo "  * rubygems"
+echo "  * All Katello and Foreman Packages"
+echo ""
+echo "Once these packages and configuration files are removed there is no going back."
+echo "If you use this system for anything other than Katello and Foreman you probably"
+echo "do not want to execute this script."
+echo ""
+read -p "Read the source for a list of what is removed.  Are you sure(Y/N)? " -n 1 -r
+echo    # (optional) move to a new line
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    echo "** cancelled **"
+    exit 1
+fi
+
+echo ""
+echo "ARE YOU SURE?: This script permanently deletes data and configuration."
+read -p "Read the source for a list of what is removed.  Type [remove] to continue? " -r
+echo    # (optional) move to a new line
+if [[ ! $REPLY = remove ]]
+then
+    echo "** cancelled **"
+    exit 1
+fi
+
+CONFIG_FILES=(
+    /etc/pulp/
+    /usr/share/katello-installer/
+    /var/lib/puppet/
+    /var/lib/mongodb/
+    /var/lib/pulp/
+    /etc/httpd/
+    /usr/share/pulp
+    /etc/foreman-proxy
+    /var/lib/foreman-proxy
+    /etc/katello-installer
+    /etc/foreman-proxy/
+    /etc/puppet/environments
+    /etc/pki/katello-certs-tools
+)
+
+LOG_FILES=(
+    /var/log/katello-installer/
+    /var/log/pulp/
+    /var/log/httpd/
+    /var/log/mongodb/
+    /var/log/foreman-proxy
+)
+
+RPMS=(
+    `rpm -qa | grep puppetlabs-release`
+    `rpm -qa | grep foreman-release`
+    `rpm -qa | grep foreman-client`
+    `rpm -qa | grep foreman-proxy`
+    `rpm -qa | grep katello`
+    `rpm -qa | grep ^pulp`
+    `rpm -qa | grep ^python-pulp`
+    `rpm -qa | grep ^pulp-`
+    `rpm -qa | grep mongo`
+    `rpm -qa | grep ^mod_`
+    `rpm -qa | grep ^rubygem`
+    `rpm -qa | grep ^ruby193`
+    `rpm -qa | grep ^foreman`
+    `rpm -qa | grep ^qpid`
+    `rpm -qa | grep ^lucene`
+    `rpm -qa | grep ^python-crane`
+    `rpm -qa | grep ^python-celery`
+    `rpm -qa | grep ^python-gofer`
+    `rpm -qa | grep ^python-qpid`
+    `rpm -qa | grep ^python-kombu`
+    `rpm -qa | grep ^python-webpy`
+    `rpm -qa | grep ^python-nectar`
+    `rpm -qa | grep ^python-saslwrapper`
+    `rpm -qa | grep ^python-amqp`
+    `rpm -qa | grep ^python-billiard`
+    `rpm -qa | grep ^python-semantic-version`
+    `rpm -qa | grep ^python-requests`
+    `rpm -qa | grep ^python-isodate`
+    `rpm -qa | grep gutterball`
+    `rpm -qa | grep $HOSTNAME`
```

Comment:
It is possible that a user has a cluster where the hostnames are simply ```1``` or ```01``` and then this can match a lot more than you expect (kernel with revision 201 for example). I'd recommend looking at the full hostname.